engine/src/main/scala/cromwell/jobstore/SqlJobStore.scala

```diff
@@ -56,6 +56,7 @@ class SqlJobStore(sqlDatabase: JobStoreSqlDatabase) extends JobStore {
       case "Int" => WdlIntegerType
       case "Float" => WdlFloatType
       case "Boolean" => WdlBooleanType
+      case "File" => WdlFileType
```

Comment:
ToL: We can discuss in person- I'm curious about simpletonization/serialization of `WdlFile`, especially as there isn't currently a one type to one value-type mapping.

`WdlFileType` (currently) has `WdlSingleFile` and `WdlGlobFile`. I'm not sure how knowing one has a `WdlFileType` alone tells you which simpleton type one should use for a round trip.

Premature optimization sucks, but based on our early discussions yesterday, I feel like groups of files might actually be appearing soon, further complicating the one file type to N value-types mapping.